src/lib/isFloat.js

```diff
@@ -10,10 +11,10 @@ export default function isFloat(str, options) {
   }
   const value = parseFloat(str.replace(',', '.'));
   return float.test(str) &&
-    (!options.hasOwnProperty('min') || value >= options.min) &&
-    (!options.hasOwnProperty('max') || value <= options.max) &&
-    (!options.hasOwnProperty('lt') || value < options.lt) &&
-    (!options.hasOwnProperty('gt') || value > options.gt);
+    (!hasOption(options, 'min') || value >= options.min) &&
```

Comment:
Was thinking of an alternative here since this function will only be needed in `isFloat` and `isInt`, can't we just use nullish coalescing operator? It will return `null` instead of `undefined` and any `value` > null will resolve to `true`
```suggestion
    value >= (options.min ?? null) &&
```

Reply:
I suppose that back when I wrote this I didn't know about the nullish coalescing op😄. Definitely what we are searching, though.